react/w-books/src/services/userService.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiResponse } from 'apisauce';
+import api from '../config/api';
+
+export const SIGN_UP = '';
+export const LOGIN = '/sign_in';
+
+export async function postUser(action: typeof SIGN_UP | typeof LOGIN, user:any):Promise<ApiResponse<any, any>> {
+  const response = await api.post(`users${action}`, user);
+  
+  if(response.ok){
+    const headers:any = response.headers;
+    const accessToken = headers['access-token'];
+    console.log(accessToken);
```

Comment:
remove

Reply:
one of the acceptance criteria of the login card (https://trello.com/c/7xLDM2t2/58-login) is this. In the next card/PR this are changed (https://github.com/wolox-training/ava-react/pull/25)